libs/java.json

```diff
@@ -215,6 +215,8 @@
   {"id":"java.flexy-pool","imports":["com.vladmihalcea.flexypool."],"name":"Flexy Pool","repo":"vladmihalcea/flexy-pool","tags":["resilience"],"tech":["reliable-code"],"status":"model-ready"},
   {"id":"java.hystrix","imports":["com.netflix.hystrix."],"name":"Hystrix","repo":"Netflix/Hystrix","tags":["fault-tolerance"],"tech":["reliable-code"],"status":"model-ready"},
   {"id":"java.resilience4j","imports":["io.github.resilience4j."],"name":"Resilience4j","repo":"resilience4j/resilience4j","tags":["fault-tolerance"],"tech":["reliable-code"],"status":"model-ready"},
+  {"id":"java.ctre-phoenix","imports":["com.ctre.phoenix."],"name":"Phoenix","repo":"CrossTheRoadElec/Phoenix-api","tags":["os"],"tech":["robotics"],"status":"awaiting-model"},
```

Comment:
So the tags, at least the first tag is what people see on your profile in the library box. Usually, it makes sense to put something in there that people in the know are familiar with. I am not a robotics expert, but would 'can-bus' be a reasonable tag? or, more generic 'toolkit'?